server/plugin.go

```diff
@@ -363,7 +367,7 @@ func (p *Plugin) OnActivate() error {
 			func() []string { return strings.Split(p.configuration.EnabledTeams, ",") },
 			func() []byte { return []byte(p.configuration.EncryptionKey) },
 		)
-		p.store = timerlayer.New(store, p.metricsService)
+		p.store = timerlayer.New(store, p.GetMetrics())
```

Comment:
Maybe we can get rid of the whole timerlayer if there is no metrics.

Reply:
We could, but feels like additional complexity, and makes it impossible to support toggling metrics. (It's actually already not supported anyway, but I digress.)